ouroboros-consensus/test-util/Test/Util/TestBlock.hs

```diff
@@ -231,9 +235,10 @@ instance UpdateLedger TestBlock where
   data LedgerState TestBlock =
       TestLedger {
           -- The ledger state simply consists of the last applied block
-          lastApplied :: (Point TestBlock, ChainHash TestBlock)
+          lastAppliedPoint :: !(Point TestBlock)
+        , lastAppliedHash  :: !(ChainHash TestBlock)
```

Comment:
Made these two separate fields, so that we could make both strict. This actually simplified the rest of this module also.